agc/anal.c

```diff
@@ -0,0 +1,166 @@
+#include <stdbool.h>
```

Comment:
Those includes should be already included by rtypes or rutil which is also included from ranal